e2e_test/js-tests/test_ethers_tx.mjs

```diff
@@ -0,0 +1,52 @@
+import { assert } from 'chai';
+import 'mocha';
+import { ethers } from "ethers";
+
+const provider = new ethers.JsonRpcProvider(process.env.ETH_RPC_URL);
+const signer = new ethers.Wallet(process.env.ACC_PRIVKEY, provider);
+
+describe('ethers.js send tx', () => {
+	it('send basic tx and check receipt', async () => {
+    const tx = await signer.sendTransaction({
+      to: '0x00000000000000000000000000000000DeaDBeef',
+      value: 1
+    });
+    const receipt = await tx.wait();
+  }).timeout(10_000);
+});
```

Comment:
Weird formatting in this particular block but formatting in this file differs from the other .mjs file, so worth unifying.

Reply:
Ah yes, I messed on in this place when converting it from a plain script to a Mocha test. I ran a formatter over all files and left a Makefile with the format command.